src/_core/components/App/AppContainer.js

```diff
@@ -74,6 +75,9 @@ export class AppContainer extends Component {
         // Perform initial browser functionality check
         this.props.checkBrowserFunctionalities();
 
+        // prep the default projection for the application
```

Comment:
Interesting, no other obvious place to put this?

Reply:
not that I could easily think of, but I am open to suggestions